test/parallel/test-buffer-concat.js

```diff
@@ -98,3 +99,16 @@ assert.deepStrictEqual(
 assert.deepStrictEqual(Buffer.concat([new Uint8Array([0x41, 0x42]),
                                       new Uint8Array([0x43, 0x44])]),
                        Buffer.from('ABCD'));
+
+// Test its possible to concat buffers where the total length of the result in bytes
+// is greater than 2^32.
```

Comment:
```suggestion
// Ref: ...
```

Reply:
Comment is replaced with a reference to the [issue](https://github.com/nodejs/node/issues/55422) as of commit [5ca703f](https://github.com/nodejs/node/pull/55492/commits/5ca703f01e7aa41eef7e3c30ac821303293cd019).